components/02-molecules/menu/_menu-item.twig

```diff
@@ -33,18 +33,18 @@
         'aria-current': 'page',
       }) %}
     {% endif %}
-    {% if directory %}
+    {% if directory and not menu__contextual %}
```

Comment:
@callinmullaney The breadcrumbs don't return the render array like the main nav and stuff do. So this was failing to work in Drupal for the breadcrumbs.

Reply:
This makes sense when clarifying things with Marc and the breadcrumb adjustments we're make in the yale_core custom module.